Styles sheet Community HTML CSS.css

```diff
@@ -0,0 +1,6 @@
+html {
```

Comment:
I would call this file ```style.css```
I think that's more conventional to how devs usually start out these sort of projects.

Reply:
Just renamed.  

Question:  How do you distinguish that style sheet, at a glance, from the many others one creates?   Is it common practice to create a separate directory for every project?  I currently house all my html files and css files in the two separate directories- one for markup and one for style sheets.  I think this is a little cumbersome.

Reply:
So this project, if you cloned it, would be in a folder called ```community-html-css-page```.  Within this folder, you would have an index.html, style.css, and probably a main.js. And since they are all in the same folder (or even in subfolders) you can reference them relatively. Later if you needed to create a styles folder and put a second stylesheet there you could adjust accordingly. But on initial creation, these three main files keep it simple to start.

Reply:
Oh and when you rename, you'll need to push that change up.